conf/nginx/nginx.conf.jnj

```diff
@@ -36,6 +36,8 @@ http {
         # This header must be set only for HTTPS
         add_header Strict-Transport-Security "max-age=63072000; preload";
 
+        real_ip_header X-Forwarded-For;
```

Comment:
We will also need to patch the `7443` port as well. That's the one being used by the ELB endpoint. So I'm not sure what effect it's going to be when we remove `proxy_protocol` and replace it with `X-Forwarded-For` some proxies support the proxy protocol header

Reply:
We'll likely need to turn it off on ELB, as nginx probably won't accept connections using a non proxy protocol server directive.